package.json

```diff
@@ -38,7 +38,7 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@graphprotocol/graph-cli": "^0.73.0",
+    "@graphprotocol/graph-cli": "^0.90.0",
```

Comment:
I just updated this in latest main so I don't have to review the changes on `yarn.lock`, can you rebase and drop your changes on both package.json and yank.lock plz 🙏

Reply:
Done

Reply:
`yarn.lock` changes still left

Reply:
Sorry, copy pasted now.